src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -59,6 +68,62 @@ internal void InitializeLicense(string licenseText)
             license = foundLicense;
         }
 
+        static bool FirstTimeUser()
+        {
+            // Check for the presence of HKCU:Software/NServiceBus
+            using (var regNsbRoot = Registry.CurrentUser.OpenSubKey(@"Software\NServiceBus"))
+            {
+                if (regNsbRoot == null)
+                {
+                    //Check for the presence of HKCU:SOFTWARE\ParticularSoftware
+                    using (var regParticularRoot = Registry.CurrentUser.OpenSubKey(@"Software\ParticularSoftware"))
+                    {
+                        if (regParticularRoot == null)
+                        {
+                            return true;
+                        }
+                    }
+                }
+            }
+            return false;
+        }
+
+        /// <summary>
+        /// Adding the async void to explicitly have a fire and forget model for invoking the web api.
+        /// </summary>
+        async void TrackFirstTimeUsageEvent()
+        {
+            // Get the current version of NServiceBus that's being used
+            var version = FileVersionRetriever.GetFileVersion(GetType());
+
+            // Report first time usage metric
+            Logger.InfoFormat("Reporting first time usage and version information to www.particular.net. For more information, see: http://particular.net/licenseagreement. The first time usage is a one time tracking event. This code will not be executed in production servers. It is executed only once for the first time when run in an interactive debugging mode.");
```

Comment:
@timbussmann - Although the privacy policy will cover it in more detail, I think we should as well. I'll make the change.